mis_builder/models/expression_evaluator.py

```diff
@@ -6,12 +6,7 @@
 
 class ExpressionEvaluator(object):
     def __init__(
-        self,
-        aep,
-        date_from,
-        date_to,
-        additional_move_line_filter=None,
-        aml_model=None,
```

Comment:
Please keep the trailing commas as they are, so there are no needless diff. I recommend you install the latest version of black, and/or run `pre-commit run -a` locally to apply the same formatters and linters than the CI.